--- a/script/config.w32.phpize.in	Wed Mar 18 08:33:59 2015
+++ b/script/config.w32.phpize.in	Thu Apr 16 18:01:44 2015
@@ -45,34 +45,6 @@
 // we don't want to define LIB, as that will override the default library path
 // that is set in that env var
 PATH_PROG('lib', null, 'MAKE_LIB');
-if (!PATH_PROG('bison')) {
-	ERROR('bison is required')
-}
-
-// There's a minimum requirement for re2c..
-MINRE2C = "0.13.4";
-
-RE2C = PATH_PROG('re2c');
-if (RE2C) {
-	var intvers, intmin;
-	var pattern = /\./g;
-
-	RE2CVERS = probe_binary(RE2C, "version");
-	STDOUT.WriteLine('  Detected re2c version ' + RE2CVERS);
-
-	intvers = RE2CVERS.replace(pattern, '') - 0;
-	intmin = MINRE2C.replace(pattern, '') - 0;
-
-	if (intvers < intmin) {
-		STDOUT.WriteLine('WARNING: The minimum RE2C version requirement is ' + MINRE2C);
-		STDOUT.WriteLine('Parsers will not be generated. Upgrade your copy at http://sf.net/projects/re2c');
-		DEFINE('RE2C', '');
-	} else {
-		DEFINE('RE2C_FLAGS', '');
-	}
-} else {
-	STDOUT.WriteLine('Parsers will not be regenerated');
-}
 PATH_PROG('zip');
 PATH_PROG('lemon');
 
--- a/script/phpize.js	Fri Mar 20 00:12:13 2015
+++ b/script/phpize.js	Thu Apr 16 18:02:52 2015
@@ -312,7 +312,6 @@
 // Pull in code for the base detection
 modules = file_get_contents(PHP_DIR + "/script/config.w32.phpize.in");
 
-C.WriteLine("ARG_ENABLE('debug', 'Compile with debugging symbols', \"no\");");
 find_config_w32(".");
 
 // Now generate contents of module based on MODULES, chasing dependencies
